packages/expo-dev-launcher/expo-dev-launcher.podspec

```
@@ -34,6 +34,15 @@ Pod::Spec.new do |s|
 
   # Swift/Objective-C compatibility
   s.pod_target_xcconfig = { "DEFINES_MODULE" => "YES" }
+  dev_launcher_url = ENV['EX_DEV_LAUNCHER_URL'] || ""
+  if dev_launcher_url != ""
+    escaped_dev_launcher_url = Shellwords.escape(dev_launcher_url).gsub('/','\\/')
+    s.pod_target_xcconfig = {
+      'DEFINES_MODULE' => 'YES',
+      'OTHER_CFLAGS[config=Debug]' => "$(inherited) -DEX_DEV_LAUNCHER_URL=\"\\\"" + escaped_dev_launcher_url + "\\\"\""
```

Might be a little simpler to use single quotes for less escaping :)
```suggestion
      'OTHER_CFLAGS[config=Debug]' => '$(inherited) -DEX_DEV_LAUNCHER_URL="\\"' + escaped_dev_launcher_url + '\\""'
```